draft-ietf-teep-architecture.md

```diff
@@ -331,8 +343,8 @@ all components are further explained in the following paragraphs.
 
 ~~~~
    +-------------------------------------------+
-   | Device                                    |
-   |                          +--------+       |        TA Developer
```

Comment:
FYI, line 335 was incorrect and didn't match the text under the diagram which had "TA Signer", so that should be fixed regardless.